src/services/Frontline.js

```diff
@@ -124,6 +135,111 @@ class Frontline {
       }
     }
   }
+
+  /**
+   * Load Infinity
+   */
+
+  httpRequestWrapper = (method, URL) => {
+    return new Promise((resolve, reject) => {
+      const xhr_obj = new XMLHttpRequest();
```

Comment:
I think its save to just use `fetch()` nowadays.

The only reason to use good ol' XHR are features not supported by `fetch()` yet (e.g. upload progress).

Reply:
Yeah, I considered the `fetch()` but I see it's not supported in some browsers like IE, really old chrome,...
But I think it's fine, I will move this to use `fetch()`.